components/canvas/canvas_data.rs

```diff
@@ -23,6 +23,7 @@ use webrender_api::units::RectExt as RectExt_;
 /// further operations to it in device space. When it's time to
 /// draw the path, we convert it back to userspace and draw it
 /// with the correct transform applied.
+/// TODO: De-abstract now that Azure is removed?
```

Comment:
I can remove these comments or change the format, I don't know what the plan is here.

Reply:
Is de-abstracting these things also considered part of the issue? It would be straightforwards, I think, I just didn't want to assume it was desired immediately

Reply:
Merging the code removing is a good step even if a now-unnecessary abstraction is still there. Leaving a TODO or FIXME comment is fine.

Reply:
@pcwalton Do you think we should keep this abstraction around in case we want to add a Pathfinder backend at some point?